docs/src/user_guide/step3_input_data.md

```diff
@@ -0,0 +1,56 @@
+# Step 3: Preparing the input data
+
+As mentioned before, the input data can be classified into two types:
+
+ - Meteorological forcing: maps with timeseries for each model pixel, with values for
+   precipitation, temperature, and potential evaporation. This data should be provided as a
+   three-dimensional dataset, with the `x`, `y` and `time` dimensions. 
+ - Static maps.
+
+
+## Meterological data
+
+Meteorological data is provided as a single NetCDF file, with several variables containing
+the forcing data for precipitation, temperature and potential evaporation. The code snippet
+below shows the contents of the example file (downloaded [here](@ref wflow_sbm_data)), and
+opened with `xarray` in Python. As can be seen, each variable consists of a
+three-dimensional dataset (`lon`, `lat`, and `time`), and each timestep consists of a
+two-dimensional map with values at each gridcell. Only values within the basin are required.
+
+```
+<xarray.Dataset>
```

Comment:
I think it would be better to show here output from Julia instead of Python, after all, this is a Julia package using NCDatasets to read and write NetCDF files.

Reply:
Changed the NetCDF information to Julia's `ncinfo` output

Reply:
Changed this to `NCDatasets` since we use this package in Wflow.